lib/transaction/trace/segment.js

```diff
@@ -233,24 +226,30 @@ TraceSegment.prototype._isEnded = function _isEnded() {
 /**
  * Add a new segment to a scope implicitly bounded by this segment.
  *
- * @param {string} childName New human-readable name for the segment.
- * @param recorder
+ * @param config.name
+ * @param config.collect
+ * @param config.traceStacks
+ * @param config.root
+ * @param root0
+ * @param root0.config
+ * @param root0.name
+ * @param root0.collect
+ * @param root0.traceStacks
+ * @param root0.root
```

Comment:
Types need to be added. There's another function several files up that only has `root0` defined in the doc block. Based upon this second reference, it seems like there should be a `@typedef` added somewhere (top of this file?) so that it can be easily referenced across the code base.